### PR TITLE
BZ1902087: Swapped mysql example for nodejs

### PR DESCRIPTION
**Pull request by ctauchen**
I changed the example app from mySQL to NodeJS.

https://bugzilla.redhat.com/show_bug.cgi?id=1902087

Applies to 4.6+

https://deploy-preview-36417--osdocs.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html
https://deploy-preview-36417--osdocs.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer.html

### CODE REVIEW

File: modules/nw-exposing-service.adoc [hunk 1]
```diff
@@ -25,67 +25,54 @@ $ oc project myproject
...
-.Example output
+.. Use cURL to check that the host responds to a GET request.
++
 [source,terminal]
 ----
-Enter password:
-Welcome to the MariaDB monitor.  Commands end with ; or \g.
-
-MySQL [(none)]>
+$ curl nodejs-ex-myproject.example.com
```
Comment (lihongan) [inline modules/nw-exposing-service.adoc:62]: or use the curl command with `-I` option, see
```
$ curl http://nodejs-ex-myproject.example.com -I
HTTP/1.1 200 OK
...
```
The response status code 200 means the application is accessible from outside the cluster.

Comment (ctauchen) [reply modules/nw-exposing-service.adoc]: That's a good suggestion -- thanks! I've made the change.

**Comment by lihongan** [comment]
LGTM
Thank you @ctauchen

**Review by adambkaplan — APPROVED** [review]
/lgtm

File: modules/nw-exposing-service.adoc [hunk 2]
```diff
@@ -25,67 +25,57 @@ $ oc project myproject
...
 ----
-route "mysql-80-rhel7" exposed
+route.route.openshift.io/nodejs-ex exposed
 ----
-
-. Use a tool, such as cURL, to make sure you can reach the service using the
-cluster IP address for the service:
++
+. To verify that the service is exposed, you can use a tool, such as cURL, to make sure the service is accessible from outside the cluster.
++
```
Comment (adellape) [inline modules/nw-exposing-service.adoc:43]: The two `+` continuations here (line 41 and 43) aren't actually needed.

Same down on line 57.

File: modules/nw-exposing-service.adoc [hunk 3]
```diff
@@ -25,67 +25,57 @@ $ oc project myproject
...
+.. Use the `oc get route` command to find the route's host name.
```
Comment (adellape) [inline modules/nw-exposing-service.adoc:44]: Colon leading into command.

File: modules/nw-exposing-service.adoc [hunk 4]
```diff
@@ -25,67 +25,57 @@ $ oc project myproject
...
+NAME        HOST/PORT                        PATH   SERVICES    PORT       TERMINATION   WILDCARD
+nodejs-ex   nodejs-ex-myproject.example.com         nodejs-ex   8080-tcp                 None
 ----
 +
-The examples in this section uses a MySQL service, which requires a client
-application. If you get a string of characters with the `Got packets out of order`
-message, you are connected to the service.
-+
-If you have a MySQL client, log in with the standard CLI command:
+.. Use cURL to check that the host responds to a GET request.
```
Comment (adellape) [inline modules/nw-exposing-service.adoc:58]: Same re: colon.

File: modules/nw-exposing-service.adoc (hunk 2 above)
Comment (ctauchen) [reply modules/nw-exposing-service.adoc:43]: Thanks -- all fixed.

File: modules/nw-exposing-service.adoc (hunk 4 above)
Comment (ctauchen) [reply modules/nw-exposing-service.adoc:58]: Thanks, changed now.

File: modules/nw-exposing-service.adoc (hunk 3 above)
Comment (ctauchen) [reply modules/nw-exposing-service.adoc:44]: Fixed.

**Comment by ctauchen** [comment]
@adellape Changes made. If you're happy, please merge and cherrypick for 4.6 +. Please leave any manual cherrypicking to me. Thanks!

**Pull request merged**